test/e2e/volumes.go

```diff
@@ -169,6 +169,11 @@ func volumeTestCleanup(client *client.Client, config VolumeTestConfig) {
 		expectNoError(err, "Failed to delete client pod: %v", err)
 	}
 
+	// Since volume clean up and umount happens in the background.
+	// It is possible that the volume server container is stopped before umount kicks in.
+	// In such case, some volume types (like NFS) is unable to umount the volume.
+	// To work around this issue, give a pause here.
+	// Ideally the pause should be longer than the kubelet's sync timeout.
```

Comment:
i think we can simplify this with `This sleep prevents a client from failing to unmount a volume, see issue #abcd`